src/_data/books-dart3.yml

```
@@ -0,0 +1,8 @@
+- title: 'Dart Apprentice'
+  authors:
+    - Jonathan Sande
+    - Matt Galloway
+  publisher: raywenderlich
+  cover: dart-apprentice.png
+  link: "https://www.raywenderlich.com/books/dart-apprentice/v1.1/"
```

Could you remove the version from this so that the link always leads to the latest version?

```suggestion
  link: "https://www.raywenderlich.com/books/dart-apprentice"
```